pkg/k8s/k8s.go

```diff
@@ -68,3 +68,30 @@ func CreateNamespaceIfNotExists(kubeconfigPath, namespace string) error {
 	}
 	return nil
 }
+
+func PrintLoadBalancerServices(kubeconfigPath, namespace string) (hostName string, err error) {
+	// Get all services in the namespace
+	var externalIP string
+	clientset, err := GetK8SClient(kubeconfigPath)
+	if err != nil {
+		return "", err
+	}
+
+	services, err := clientset.CoreV1().Services(namespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	// Iterate through the services and find those of type LoadBalancer
+	for _, service := range services.Items {
```

Comment:
use specific service to get load balancer host, configure service name in config.go